ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -257,27 +265,29 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
 
     this.addressErrors$ = this.state$.pipe(
       pluck('fieldErrors'),
-      distinctUntilChanged((x, y) => util.isEqual(x, y)),
       shareReplay()
     );
 
     this.suggestionInfo$ = this.state$.pipe(
-      map(state => state.showSuggestion && !this.readonly ? { suggested: state.suggestedAddress, entered: state.enteredAddress } : null),
+      map(state => state.showSuggestion && !this.readonly ?
+                {suggested: state.suggestedAddress,
+                  entered: state.enteredAddress,
+                  warnings: state.warnings} : null),
       shareReplay(1)
     );
 
+    this.errorMessagesToDisplay$ = this.state$.pipe(
+        map(state => state.formErrorMessages.concat(state.formWarningMessages)),
+        distinctUntilChanged((x, y) => util.isEqual(x, y)),
+        shareReplay(1)
+    );
+
```

Comment:
Combine warnings and errors for display. Choosing to display them using the validation message component.

Reply:
Cool! I think as long as it's not in red, then it should be fine group together the warnings and errors.